tests/integration/servicemesh/managingroutes/main_test.go

```diff
@@ -160,19 +331,27 @@ func getPodName(ctx framework.TestContext, namespace, appName string) (string, e
 	return pods.Items[0].Name, nil
 }
 
-func applyGatewayOrFail(ctx framework.TestContext, ns string, hosts ...string) {
+const labelSetA string = ` 
+    a: "a"
+`
+
+const labelSetB string = `
+    b: "b"
+`
+
+func applyGatewayOrFail(ctx framework.TestContext, ns, name, labels string, hosts ...string) {
 	// retry because of flaky validation webhook
 	retry.UntilSuccessOrFail(ctx, func() error {
-		return ctx.ConfigIstio().EvalFile(ns, map[string][]string{"hosts": hosts}, gatewayTmpl).Apply()
+		return ctx.ConfigIstio().EvalFile(ns, map[string]interface{}{"hosts": hosts, "name": name, "labels": labels}, gatewayTmpl).Apply()
```

Comment:
Parameters "name" and "labels" are not interpolated in the template. I guess this is the reason why the tests fail.